core-data/reducer.js

```diff
@@ -0,0 +1,13 @@
+const reducer = ( state, action ) => {
```

Comment:
- Maybe fine as a demo / proof-of-concept, but this reducer shape certainly won't scale to accommodate more data.
- We should want some documentation when we do flesh it out.